### PR TITLE
feature(#970) : force internal comments for KtCommentInput (and KtComment)

### PR DESCRIPTION
**Pull request by RueRivoli**
This merge request intends to block comments into the internal mode meaning that the user cannot switch to public.
This prop is meaningful only if the property allowInternal is true.

### CODE REVIEW

File: packages/kotti-ui/source/kotti-comment/KtComment.vue [hunk 1]
```diff
@@ -25,7 +25,7 @@
 			/>
 			<KtCommentInput
 				v-if="userToReply"
-				v-bind="{ isInternal, placeholder, tabIndex, userAvatar }"
+				v-bind="{ forceInternal, isInternal, placeholder, tabIndex, userAvatar }"
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-comment/KtComment.vue:28]: Where does this variable come from?

**Review by Isokaeder — APPROVED** [review]
FT okay

**Review by FlorianWendelborn — APPROVED** [review]
LGTM

**Comment by Isokaeder** [comment]
This resolves #970

**Pull request merged**